templates/default/yarn-site.xml.erb

```diff
@@ -62,97 +93,183 @@
     <description>The actual address the server will bind to. If this optional address is set, the RPC and webapp servers will bind to this address and the port specified in yarn.nodemanager.address and yarn.nodemanager.webapp.address, respectively. This is most useful for making NM listen to all interfaces by setting to 0.0.0.0.</description>
   </property>
 
+  <property>
+    <name>yarn.log-aggregation.retain-seconds</name>
+    <value><%= node['hops']['yarn']['log_retain_secs'] %></value>
+    <description>Default time (in seconds) to retain log files in HDFS.
+    Only applicable if log-aggregation is disabled.</description>
```

Comment:
just a nit pick: the description here is wrong. `How long to keep aggregation logs before deleting them. -1 disables. Be careful set this too small and you will spam the name node.` is the correct description